libraries/AP_InertialSensor/AP_InertialSensor_SCHA63T.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  the BMI055 is unusual as it has separate chip-select for accel and
+  gyro, which means it needs two SPIDevice pointers
+ */
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+#include "AP_InertialSensor.h"
+#include "AP_InertialSensor_Backend.h"
+
+static constexpr int16_t combine(uint8_t msb, uint8_t lsb)
```

Comment:
please put this in the cpp, so it isn't exposed to other cpp files

Reply:
Thank you.
I changed it.